.github/dependabot.yml

```diff
@@ -15,4 +15,4 @@ updates:
       prefix: "[No Ticket]"
     ignore:
       - dependency-name: "org.springframework.boot:spring-boot-gradle-plugin"
-        versions: ["3.x"]
+        update-types: ["version-update:semver-major"]
```

Comment:
Syntax from https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore